src/buza/models.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import unicode_literals
+
+from django.contrib.auth.base_user import AbstractBaseUser
+from django.contrib.auth.models import PermissionsMixin
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from project.boards.models import Board
+
+from .managers import UserManager
+
+
+# Models will be added to the db
+
+
+class BuzaUser(AbstractBaseUser, PermissionsMixin):
+    # authentication feilds
+    email = models.EmailField(
+        _('email address'),
+        unique=True,
+        blank=True,
+    )
+    phone = models.CharField(
+        _('phone number'),
+        blank=True,
+        unique=True,
+        max_length=11,
+    )
+    user_name = models.CharField(
+        _('user name'),
+        blank=True,
+        null=True,
+        unique=True,
+        max_length=15,
+    )
```

Comment:
We should weigh whether we really want a separate `username` field or not… It's something Django usually assumes as a unique identifier, but it may be simpler for students to just not have this (and just use the PK as the internal unique identifier)?